openedx/features/course_experience/views/course_home_messages.py

```diff
@@ -172,7 +172,7 @@ def _register_course_goal_message(request, course):
             '{initial_tag}{goal_text}{closing_tag}'
         ).format(
             initial_tag=HTML(
-                '<button tabindex="0" aria-label="{aria_label_choice}" class="goal-option btn" '
+                '<button tabindex="0" aria-label="{aria_label_choice}" class="goal-option btn-outline-primary" '
```

Comment:
I think we need to always include `btn` as well as the more specific button class. At least, I saw problems when not doing this.

Reply:
The Bootstrap doc always includes both:

https://getbootstrap.com/docs/4.0/components/buttons/